.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
+*.po
```

Comment:
@luzfcb good point, I wanted to exclude the .mo files.

@kmmbvnr whatever makes you happy. It just makes merging multiple pull-requests very difficult.
You'll need to recreate the compiled files over and over again.
Django just changed how compiling is done in development branch. So you can stop including compiled files in your own apps.
I think it's saver to create the compiled files during distribution. Also why store compiled files in a repository? Reversion tracking is for source code ;)